frontend/components/App.js

```diff
@@ -302,23 +291,27 @@ export default class App extends React.Component {
 
           <section>
             <h1>Lumen distribution</h1>
+            <div className="mui-col-md-4">
+              <DistributionProgress horizonLiveURL={horizonLive} />
+            </div>
 
-            <h2>
-              11/4/2019: Updates to Lumen Distribution are currently paused. We will release an updated version of the Dashboard soon.
-            </h2>
+            <div className="mui-col-md-4">
+              <TotalCoins horizonURL={horizonLive} />
+            </div>
+
+            <div className="mui-col-md-4">
+              <LumensAvailable />
+            </div>
           </section>
 
           <section>
-            <h1>Featured live network nodes</h1>
+            <h1>Network Nodes</h1>
             <h2>
-              None of the following validators are recommended by Stellar
-              Development Foundation. We don't know who really controls
-              unverified nodes.
-              <br />
-              These are <u>not</u> the only nodes in the Stellar network.
-              Everyone can run a validating node.
+              View network nodes on Stellarbeat and visualize consensus.
               <br />
-              This list is purely for informational purposes.
+              <a href="https://stellarbeat.io" target="_blank">
+                Explore Nodes
+              </a>
             </h2>
             <Nodes />
```

Comment:
I think the requirement was to remove `<Nodes />` too but can be done in a separate PR.

Reply:
Removed!

Reply:
OK, if you want to do it in this PR can you also remove backend logic that updates uptime stats?

Reply:
Should I just delete the entire `backend/nodes.js` and `common/nodes.js`?

Reply:
Ahh, I think `common/nodes.js` is used to generate: https://www.stellar.org/developers/guides/nodes.html . I wonder if we want to remove this one too, @jedmccaleb?

Reply:
we do want to remove https://www.stellar.org/developers/guides/nodes.html (cc @rice2000).

Reply:
I think we should remove that @bartekn.  We can push validators to self-verify using SEP-20, and point them to stellarbeat, and we'll have better, more up to date info.